zaryabkhan-rs/FOEDAG
# Pin Assign Opt in the Add Constraint file form

> ### Motivate of the pull request
> - [ ] To address an existing issue. 
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
>- Add pin assignment option in Add_Constraint_form dialog.

> #### What does this pull request change?
>  This PR improves in the following aspects:  
>- Pin Assignment Method Selection.

> 
> 

> ### Testing Steps:
>- create_design project_1
>- Select pin option from add_constraint_form and check on the placement setting either the selection is set or not
>- pin_loc_assign_method random/in_define_order  -->  change the pin assign selection from console.

> ### Which part of the code base require a change
> 
> - [ ] Library: <Specify the library name>
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

> ### Impact of the pull request

> - [ ] Added new options to GUI for Add Constraint Form

> ### Update Screenshots

![image](https://user-images.githubusercontent.com/109368108/185140736-6eccbd4b-f4e8-4a3a-8054-f6c8e5636d99.png)
![image](https://user-images.githubusercontent.com/109368108/185140770-8a34c14c-c6c2-4d3d-9768-306f8b7dcb9a.png)
